apps/src/lib/kits/maker/toolkit.js

```diff
@@ -164,8 +164,9 @@ function confirmSupportedBrowser() {
  * @returns {Promise.<MakerBoard>}
  */
 function getBoard() {
+  const makerBoardAPI = getAppOptions().level.makerlabEnabled;
```

Comment:
Not sure where this is called, but is it safe to assume that level will always exist? Maybe safe navigate?

Reply:
I just tried locally on standalone projects with micro:bit and with CP and `level.makerlabEnabled` is assigned.
Also, do you mean to do something like this?
`const makerBoardAPI = getAppOptions().level ? getAppOptions().level.enabled : null;` ?

Reply:
Confirmed offline that yes, optional chaining.